README.md

```diff
@@ -606,6 +606,7 @@ These functions provide a very clean way to build out very simple functions and
 | `filter` | <code>((a -> Boolean) &#124; Pred a) -> m a -> m a</code> |
 | `first` | `m (a -> b) -> m (Pair a c -> Pair b c)` |
 | `fold` | `Semigroup s => m s -> s` |
+| `fork` | `Async m => (a -> _) -> (b -> _) -> () -> m a b -> _` |
```

Comment:
hmmm. thinking it is a function that is basically a `unit` with a wonderful little side-effect. But since we cannot represent "hey this is a side-effect" in a type siggy, what do you think of something like:
```haskell
Async m => (a -> ()) -> (b -> ()) -> m a b -> (() -> ())
```
We will have to do some explaining when we do the documentation. (sooo much documentation needed)

What do you think about that?